fixtures/module/module.tf

```diff
@@ -6,7 +6,15 @@ module "test_module" {
   region = "${var.region}"
   env_name = "${var.env_name}"
 
+  env_name = "${var.env_name}"
```

Comment:
env_name was duplicate and I removed that. We need the others as the module uses the terraform config files of the aws fixture. If we don't have those, we get the below error.

```
* module root:
      module test_module: required variable "build_id" not set
      module test_module: required variable "build_pipeline_name" not set
      module test_module: required variable "build_team_name" not set
      module test_module: required variable "build_name" not set
      module test_module: required variable "build_job_name" not set
      module test_module: required variable "atc_external_url" not set
  ▲ ▲ ▲ ▲ ▲ ▲ ▲ ▲ ▲ ▲ Terraform Apply ▲ ▲ ▲ ▲ ▲ ▲ ▲ ▲ ▲ ▲
  Failed To Run Terraform Apply!
```